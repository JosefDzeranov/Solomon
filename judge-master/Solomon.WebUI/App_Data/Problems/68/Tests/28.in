50 10
-622584515 226698719 799160134 883378356
270925898 -711495242 937142840 -280997595
-626949535 146500638 944435044 708349645
-535350928 282695435 -526158789 478458365
-669957800 -817982916 -316839318 -493071509
-221218325 868478429 830690935 980241546
-646047173 -515010364 -226741026 235892635
448152863 -321969339 927465571 859427750
691202750 9774922 865331199 46046867
225903149 438278876 750093290 703316520
-583034488 -267913337 173880589 -249348855
36192716 -774105166 777075319 311122213
-985014442 -661281051 -148574179 907561416
-187579088 -886881870 975710692 -505265283
-321713571 -788446497 427770850 -283209517
-698027177 535054025 -617768278 680240038
-668013454 945572022 -462512780 990147733
-726220038 -556041441 -41643315 763985811
13576290 -337117086 451283171 935522100
-611160437 -993151425 883246043 -578967738
304212374 -55200445 472775944 679962071
-105959781 133848369 948880901 723371823
53310196 -291370241 904676778 610864399
-242301876 88725091 -155683276 707023188
-873063152 580184886 36531940 973991780
-971933481 21147280 320883020 476248017
-760070468 -739984430 -141408175 322154891
-75481809 136145669 890891443 420909000
852020597 146829781 980344702 260239760
-697815839 -466033767 -533771411 -219074272
418776205 -442302667 650236817 437411427
-885501926 -600345604 -31429435 -275366427
-639626626 -579553665 -228230448 240344790
-538921070 -930020785 799147901 -726456600
-592197075 -476690804 81623853 -414942893
-883879696 347908504 -710011261 876045058
71441395 -84093078 401028672 782241261
13042326 -420735807 459005217 141416079
704748221 -801690839 803391469 535257358
-684048339 -238028771 135118205 -69925251
-986360172 56549982 227069808 269752962
-837025364 -977977670 542842436 -399749243
752125417 -766460738 847418552 582821944
-631476644 33023938 224225702 466404491
-756282068 -809760317 449603398 -325883003
-990289817 -959344882 487588301 -802249633
-617360443 376697167 790648756 524486478
-620421754 127930363 502940337 200005925
-538398962 -847482636 38453864 -603765691
364518303 -109666596 890956982 202303082
-9906918 10590497
10379616 14769432
-23511810 3565922
-32945051 11049374
-36260404 4247293
-38363231 9284820
-12967262 12986929
-39973234 7480128
1498011 15905050
-14421960 5249096
