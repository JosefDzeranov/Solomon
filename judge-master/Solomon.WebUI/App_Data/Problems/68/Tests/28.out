YES
2
0 1 -1
-9906918 1 1
YES
2
0 1 1
10379616 1 -1
YES
2
0 1 -1
-23511810 1 1
YES
2
0 1 -1
-32945051 1 1
YES
2
0 1 -1
-36260404 1 1
YES
2
0 1 -1
-38363231 1 1
YES
2
0 1 -1
-12967262 1 1
YES
2
0 1 -1
-39973234 1 1
YES
2
0 1 1
1498011 1 -1
YES
2
0 1 -1
-14421960 1 1
